tableauserverclient/models/schedule_item.py

```diff
@@ -0,0 +1,239 @@
+import xml.etree.ElementTree as ET
+from .interval_item import *
+from .property_decorators import property_is_enum, property_not_nullable
+from .. import NAMESPACE
+
+
+class ScheduleItem(object):
+    class Type:
+        Extract = "Extract"
+        Subscription = "Subscription"
+
+    class ExecutionOrder:
+        Parallel = "Parallel"
+        Serial = "Serial"
+
+    class State:
+        Active = "Active"
+        Suspended = "Suspended"
+
+    def __init__(self, name, priority, schedule_type, execution_order, interval_item):
+        self._created_at = None
+        self._end_schedule_at = None
+        self._execution_order = None
+        self._frequency = None
```

Comment:
I just confirmed I can remove that attribute entirely, tests pass (other than the ones that check for it explicitly), and the sample still runs... should I nix it entirely?